caikit/runtime/service_generation/protoable.py

```diff
@@ -91,6 +99,30 @@ def handle_protoables_in_union(arg_type: Type) -> Type:
     log.debug("Skipping non-protoable argument type [%s]", arg_type)
 
 
+def get_union_list_type(field_name: str, union_protoables: List) -> Type[DataBase]:
+    """Create a union from list type objects"""
+    common_dm_package = caikit.interfaces.common.data_model
+    param_list = []
+    for arg in union_protoables:
+        if get_origin(arg) == list:
+            arg_type = get_args(arg)[0]
+            arg_name = f"{arg_type.__name__.capitalize()}Sequence"
+            if not hasattr(common_dm_package, arg_name):
+                raise AttributeError(
+                    f"Unable to find {arg_name} in {common_dm_package}"
+                )
+            do = getattr(common_dm_package, arg_name)
```

Comment:
Is `do` here `data object`? Total NIT, but it would be better to have a more descriptive name here, or alternately just smash this into an inline call

```py
data_obj = getattr(common_dm_package, arg_name, None)
if dat_obj is None:
    raise AttributeError(...)
param_list.append(
    Annotated[data_obj, ...]
)
```